### PR TITLE
Theme Persist

### LINKED ISSUE #330: Theme Persist Support
The ability to set/get and persist the default values for each widget.

### PR DESCRIPTION
**Pull request by GordonSmith**
Add ability to change default values of published properties.

Fixes #330

Signed-off-by: Gordon Smith <gordon.smith@lexisnexis.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jbrundage Please Review

**Comment by jbrundage** [comment]
I'll be reviewing and working with this code change today and tomorrow

File: src/c3chart/Area.js [hunk 1]
```diff
@@ -8,11 +8,11 @@
 }(this, function (CommonND) {
     function Area(target) {
         CommonND.call(this);
-        this._class = "c3chart_Area";
 
         this._type = "area";
     };
     Area.prototype = Object.create(CommonND.prototype);
+    Area.prototype._class += " c3chart_Area";
```
Comment (mlzummo) [inline src/c3chart/Area.js]: += is so that Common.js and CommoND.js can now have there own classes added to the div container?

Comment (GordonSmith) [inline src/c3chart/Area.js]: Yes - The class for the Area chart will now be ".common_Widget .c3chart_Common .c2chart_CommonND .c3chart_Area" - in the few instances we specifically needed this previously, the class was appened manually in the "enter" method.

**Comment by mlzummo** [comment]
All is well ... LGTM

**Comment by GordonSmith** [comment]
@jbrundage - Have you had a chance to work with it yet?

**Comment by GordonSmith** [comment]
Rebased for merge.

**Pull request merged**